Userland/Services/WindowServer/Button.cpp

```diff
@@ -62,22 +62,36 @@ void Button::on_mouse_event(const MouseEvent& event)
 {
     auto& wm = WindowManager::the();
 
-    if (event.type() == Event::MouseDown && event.button() == MouseButton::Left) {
+    if (event.type() == Event::MouseDown && event.button() > MouseButton::None && event.button() < MouseButton::Back) {
```

Comment:
This comparison feels really weird and depends on the enum values being in a certain order.
Let's instead test for each of the buttons we want explicitly (i.e Left, Right and Middle)